src/features/run_test_cases/run_solution.ts

```diff
@@ -90,6 +90,17 @@ export const runTestsWithTimeout = async (
                 ? "java.exe"
                 : "java";
             break;
+        case CompilationLanguages.rust:
```

Comment:
This entire block of code is not necessary. You can move  "case CompilationLanguages.rust:" to below the c/cpp/haskell case block

Reply:
I have made the necessary adjustments.
https://github.com/IEEE-NITK/CodePal/commit/a2a2175f501b0e68c9ea5f4b259cf7347a313a04